content/src/fieldConfig/config.json

```diff
@@ -319,6 +319,11 @@
     "savingText": "Saving",
     "savedText": "Saved 1 minute ago"
   },
+  "seoAndAccessibilityPageTitles": {
+    "homePageTitle": "Home",
+    "dashboardPageTitle": "Dashboard",
+    "profilePageTitle": "Business Profile"
```

Comment:
`seoAndAccessibilityPageTitles.profilePageTitles` seems unnecessarily redundant. I would either put these under their appropriate pages, like so:

- `pagesMetadata.home.title`
- `pagesMetadata.dashboard.title`
- `pagesMetadata.profile.title`

Or if you think the better thing to do would be to keep the key `seoAndAccessibilityPageTitles`, I'd recommend this approach:

- `seoAndAccessibilityPageTitles.home`
- `seoAndAccessibilityPageTitles.dashboard`
- `seoAndAccessibilityPageTitles.profile`

I prefer the first approach, though, since it gives us room to expand page metadata in the future.

Reply:
Will modify accordingly after lunch